server/src/main/java/org/eclipse/openvsx/UserAPI.java

```diff
@@ -203,6 +203,7 @@ public List<ExtensionJson> getOwnExtensions() {
                     var json = latest.toExtensionJson();
                     json.preview = latest.isPreview();
                     json.active = latest.getExtension().isActive();
+                    json.potentiallyMalicious = latest.isPotentiallyMalicious();
```

Comment:
This is going to be used in the webui?

Reply:
Not planned yet, therefore I removed it again. We can add it back if there ever is a plan to show it in the webui.